tyssue/core/history.py

```diff
@@ -141,6 +144,9 @@ def retrieve(self, time):
         If a specific dataset was not recorded at time time, the closest record before that
         time is used.
         """
+        if time > self.datasets[element]["time"].values[-1]:
```

Comment:
replace `element` with `"vert"` to fix the unbound local error failing in travis ;)